detection-rules/spam_campaign_excessive_display_text_with_keywords.yml

```
@@ -0,0 +1,17 @@
+name: "Spam: Campaign with excessive display-text and keywords found"
+type: "query"
+description: |
+  Detects affiliate marketing spam where any link contains display-text greater than 3000 chars and specific values found. 
+source: |
+  type.inbound
+  and length(body.links) > 0
+  and (
+    any(body.links, length(.display_text) >3000)
+  )
+  and any(
+    body.links, regex.icontains(.display_text, '(\bPassword:)', 'Hi.{0,5}Welcome\b')
+    )
+severity: "low"
+tags:
+  - "Suspicious Link"
```

```suggestion
  - "Suspicious link"
```

[lowercase in other tags](https://github.com/sublime-security/sublime-rules/search?q=%22suspicious+link%22) (would show up as a dupe in the dash)